Generate_Client/generate.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+
+origin="$(pwd)"
```

Comment:
I recommend using a construct like:
```bash
SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
```
to "anchor" the script within the source tree. Then using the SCRIPT_DIR you can change paths relative to where the script is within the source tree.

Reply:
Ack